plugins/rake-fast/rake-fast.plugin.zsh

```diff
@@ -12,13 +12,18 @@ _rake_does_task_list_need_generating () {
 }
 
 _is_rails_app () {
-  [[ -e "bin/rails" ]] || [ -e "script/rails" ]
+  [[ -e "bin/rails" ]] || [[ -e "script/rails" ]]
 }
 
 _tasks_changed () {
-  local is_changed=1
-  for file in lib/tasks/**/*.rake; do
-    if [[ $file -nt .rake_tasks ]]; then is_changed=0; fi
+  local is_changed paths
+  is_changed=1
+  paths=(lib/tasks lib/tasks/**/*(:f))
```

Comment:
Ok found it on [Modifiers](http://zsh.sourceforge.net/Doc/Release/Expansion.html#Modifiers):

> The following f, F, w and W modifiers work only with parameter expansion and filename generation. They are listed here to provide a single point of reference for all modifiers.

> f
>     Repeats the immediately (without a colon) following modifier until the resulting word doesn’t change any more.

Still, that doesn't seem to be doing anything.